tests/Resources/ODS.robot

```diff
@@ -202,13 +202,23 @@ Verify Default Access Groups Settings
         Verify RHODS Groups Config Map Contains Expected Values   &{exp_values}
     END
 
-Disable Access To Grafana Using OpenShift Port Forwarding
+Disable Access Using OpenShift Port Forwarding
     [Documentation]   Kill process running in background based on Id
-    [Arguments]  ${PROC}
-    Terminate Process   ${PROC}
+    [Arguments]  ${port_forwarding_process}
+    Terminate Process   ${port_forwarding_process}
```

Comment:
This keyword seems to be general but then the names are oddly specific.
I also don't understand if it's supposed to be killing a process on the machine running the test or somewhere in the cluster.

Reply:
Updated